test/stmatic-test.ts

```diff
@@ -1048,6 +1046,56 @@ describe("Starting to test StMATIC contract", () => {
                 });
             }
         });
+
+        it("should not revert if a validator does not accumulate enough rewards", async () =>{
+            const numOperators = 2;
+            for (let i = 1; i <= numOperators; i++) {
+                await mint(testers[i], ethers.utils.parseEther("100"));
+                await addOperator(
+                    `BananaOperator${i}`,
+                    testers[i].address,
+                    ethers.utils.randomBytes(64)
+                );
+                await stakeOperator(i, testers[i], "100");
+            }
+            await stMATIC.setDelegationLowerBound(5);
+
+            await stMATIC.setRewardDistributionLowerBound(
+                ethers.utils.parseEther("100")
+            );
+
+            const validatorShareRewards = [1000, 10];
+            for (let i = 1; i <= numOperators; i++) {
+                await mint(testers[i], ethers.utils.parseEther("10"));
+                await submit(testers[i], ethers.utils.parseEther(String(10)));
+
+                // transfer some tokens to the validatorShare contracts to mimic rewards.
+                const rewardAmount = validatorShareRewards[i-1];
+                await mint(deployer, ethers.utils.parseEther(rewardAmount.toString()));
+                await mockERC20.transfer(
+                    await getValidatorShareAddress(i),
```

Comment:
Transfer only rewards to the first validator, and for the second one keep rewards 0